hexa/workspaces/schema/mutations.py

```diff
@@ -266,8 +265,57 @@ def resolver_join_workspace(_, info, **kwargs):
         }
 
 
+@workspace_mutations.field("resendWorkspaceInvitation")
+def resolve_resend_workspace_invitation(_, info, **kwargs):
+    request: HttpRequest = info.context["request"]
+    input = kwargs["input"]
+
+    try:
+        invitation = WorkspaceInvitation.objects.get(id=input["invitationId"])
+        if not request.user.has_perm("workspaces.manage_members", invitation.workspace):
+            raise PermissionDenied
+
+        if invitation.status == WorkspaceInvitationStatus.ACCEPTED:
+            raise AlreadyExists("Invitation already accepted.")
+
+        token = invitation.generate_invitation_token()
+        send_mail(
```

Comment:
If it's 99% the same code as the initial invite, it's better to make a function of this and call it at the different places.